### DISCUSSION TITLE
Changing error messages...

### DISCUSSION THREAD

**Discussion by roracle** [EssentialsX/Essentials#4406] [Q&A] [answered]
When doing the `/sell all` command, an error comes out saying it was unable to sell (insert custom items here).  I use Fancy Bags and Emerald Tools, and those are always triggering the error message.
I would like to know how to customize the error message to no longer show, as the message is obvious.  Players can only sell raw gold, raw iron, and raw copper (including raw block variants).  I need a way to customize this so it no longer displays when players use the command.
Thank you!

PS: I saw a video regarding changing messages, but I couldn't find the message "You are not allowed to sell these items named:" message.  I'm not sure where to go from this point.  Again, thank you for your time.

**Comment by roracle** [comment, marked as answer]
Okay I found the proper file for it, but I'm still getting "Error:" by itself. This is what needs to go away.

**Reply by Chew** [reply]
```errorWithMessage=\u00a7cError\:\u00a74 {0}```
You'd want to replace this with 
```errorWithMessage=\u00a74 {0}```
(Source https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/main/resources/messages.properties#L295)

**Reply by roracle** [reply]
Thank you, I had found it and forgot to update. Good info to add to the post for future reference though.
